clients/go/cmd/zbctl/main_test.go

```diff
@@ -118,16 +130,16 @@ func TestZbctlWithInsecureGateway(t *testing.T) {
 		})
 }
 
-func (s *integrationTestSuite) TestCommonCommands() {
+func (suite *integrationTestSuite) TestCommonCommands() {
```

Comment:
hehe, welcome to the world of go, where all variables are a single letter :joy:

Reply:
Oh is this convention?🙈😅 I hope not 😅

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#variable-names :man_shrugging: I think I made it sound worse than it is, I kind of get the reasoning, but I guess it mostly works if you're disciplined and keep your functions pretty short.